build/redeemer/Redeemer.sol

```diff
@@ -8,7 +8,7 @@ import "./Safe.sol";
 
 contract Redeemer {
   address public admin;
-  address public marketPlace;
+  MarketPlace public marketPlace;
```

Comment:
traditionally we have stored the address and hydrated the instance on demand.

this is a bigger change, and would need its own PR likely as this would suggest swivel should do this as well (storing instances vs addresses). 

let's discuss this on Monday